src/frontend/src/constant/constant.ts

```diff
@@ -1,26 +1,29 @@
+import { DelegatedReviewTypesEnum } from '../ts/enums/delegated-review-types.enum';
+import { PiaTypesEnum } from '../ts/enums/pia-types.enum';
+
 export const OtherFactor = [
   {
     label: 'Sensitive personal information',
     value: 'hasSensitivePersonalInformation',
-    tooltip: true,
+    tooltip: false,
     tooltipText: `There is no complete definition for sensitive personal information. That’s because 
     sensitivity depends on context and the type of personal information involved. Any type of personal 
     information can be sensitive in one context, and not sensitive in others. Ask your MPO for help, 
     and learn more about sensitive personal information. Ask your MPO for help to identify whether 
     personal information in your initiative is sensitive.`,
   },
-  {
-    label: 'Information Sharing Agreement',
-    value: 'hasSharingOfPersonalInformation',
-    tooltip: true,
-    tooltipText: `An information sharing agreement allows a public sector organization, such as a ministry, 
-    to share personal information with another government or government agency on a regular basis. ISAs are 
-    not necessary when one B.C. government ministry shares personal information with another.`,
-  },
+  /*{
+    //label: 'Information Sharing Agreement',
+    //value: 'hasSharingOfPersonalInformation',
```

Comment:
i guess we can delete these unuse code or add a note mentioned we will use it in the future